### PR TITLE
Fix: Android page development section (Fix horizontal scroll)

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [x] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [x] Make MR mark as ready before submitting it for review
- [x] Add images, videos, or gifs of UI
- [ ] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**

- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook (chrome, firefox, safari)
- [x] iPhone(safari)
- [ ] Available android devices (chrome, firefox)
- [x] iPad(safari)
- [ ] Tablet (chrome, firefox)

**Description:**
Add horizontal smooth scroll.
**ios-development page**
2. Fix lagging on ios-development section.
3. Fix button landing section.(appearing over header)
**Video:**

https://github.com/canopas/canopas-website/assets/98794636/5ee51fe6-a2ab-4f8b-9f0b-9544a50cd4bc

### CODE REVIEW

File: vue-frontend/src/components/android-app-development/DevelopmentSection.vue [hunk 1]
```diff
@@ -18,16 +15,88 @@
         growth and customer engagement.</span
       >
     </div>
-    <div class="swiper-content tw-mt-8">
+
+    <!--Just extra div for scrolling space-->
+    <div
+      class="sticky-parent tw-relative tw-h-[150vh] xll:tw-h-[100vh] tw-hidden md:tw-block tw-mr-[-12rem]"
```
Comment (cp-mansi-d) [inline vue-frontend/src/components/android-app-development/DevelopmentSection.vue:21]: use tw-h-screen insted of tw-h-[100vh]

File: vue-frontend/src/components/android-app-development/DevelopmentSection.vue [hunk 2]
```diff
@@ -18,16 +15,88 @@
...
+        >
+          <div
+            class="tw-relative !-tw-mr-[14.5rem] tw-mb-8 tw-mt-12 tw-hidden md:tw-block"
+          >
+            <div
+              class="tw-border-b-2 tw-border-dotted tw-border-[#000000]/[0.87] tw-text-center"
+            ></div>
+          </div>
+          <div
+            class="tw-mr-4 tw-rounded-[1.25rem] tw-px-2 tw-py-8 md:tw-w-[34.7rem] lg:tw-w-[40.75rem] xl:!tw-w-[45.75rem] tw-mx-auto"
```
Comment (cp-mansi-d) [inline vue-frontend/src/components/android-app-development/DevelopmentSection.vue:40]: follow Guidelines for css

File: vue-frontend/src/components/android-app-development/DevelopmentSection.vue [hunk 3]
```diff
@@ -18,7 +15,78 @@
...
+                class="tw-border-b-2 tw-border-dotted tw-border-[#000000]/[0.87] tw-text-center"
+              ></div>
+            </div>
+            <div
+              class="tw-absolute tw-top-[15.5%] xl:tw-top-[13.4%] xll:tw-top-[32.4%] tw-ml-[45%] xll:tw-ml-[47%] tw-hidden tw-h-20 !tw-bg-white tw-text-center md:tw-block"
+            >
+              <span
+                class="tw-text-center tw-font-inter-bold tw-text-[3.75rem] tw-leading-[5.625rem] !tw-bg-white"
+              >
+                0{{ index + 1 }}</span
```
Comment (cp-mansi-d) [inline vue-frontend/src/components/android-app-development/DevelopmentSection.vue:38]: The box is correctly centered, but these numbers do not indicate the correct center in all boxes
Android-App-Development-Company-Canopas

Comment (cp-priyanka-g) [reply vue-frontend/src/components/android-app-development/DevelopmentSection.vue:38]: which width?

File: vue-frontend/src/components/android-app-development/DevelopmentSection.vue [hunk 4]
```diff
@@ -18,7 +15,78 @@
         growth and customer engagement.</span
       >
     </div>
-    <div class="swiper-content tw-mt-8">
+    <!-- Desktop UI -->
+    <div class="sticky-parent tw-hidden md:tw-block">
+      <div class="sticky tw-sticky tw-top-0 tw-max-h-screen tw-overflow-hidden">
+        <div class="tw-flex">
+          <div
+            v-for="(item, index) in items"
```
Comment (cp-jagruti-a) [inline vue-frontend/src/components/android-app-development/DevelopmentSection.vue:23]: last box is not stopping in center
Screenshot from 2023-09-08 12-24-25

File: vue-frontend/src/components/android-app-development/DevelopmentSection.vue (hunk 3 above)
Comment (cp-mansi-d) [reply vue-frontend/src/components/android-app-development/DevelopmentSection.vue:38]: i'm not saying about width

Comment (cp-jagruti-a) [reply vue-frontend/src/components/android-app-development/DevelopmentSection.vue:38]: it is happening for all

File: vue-frontend/src/components/android-app-development/DevelopmentSection.vue [hunk 5]
```diff
@@ -18,7 +15,78 @@
         growth and customer engagement.</span
       >
     </div>
-    <div class="swiper-content tw-mt-8">
+    <!-- Desktop UI -->
+    <div class="sticky-parent tw-hidden md:tw-block">
```
Comment (cp-jagruti-a) [inline vue-frontend/src/components/android-app-development/DevelopmentSection.vue:19]: this space is too much for 4k devices
Screenshot from 2023-09-08 12-29-52

Comment (cp-priyanka-g) [reply vue-frontend/src/components/android-app-development/DevelopmentSection.vue:19]: its requires as scroll will not work

Comment (cp-jagruti-a) [reply vue-frontend/src/components/android-app-development/DevelopmentSection.vue:19]: but it is not looking good we can't take this much of space

Comment (cp-priyanka-g) [reply vue-frontend/src/components/android-app-development/DevelopmentSection.vue:19]: lets discuss this space thing with sir.

File: vue-frontend/src/components/android-app-development/DevelopmentSection.vue (hunk 3 above)
Comment (cp-priyanka-g) [reply vue-frontend/src/components/android-app-development/DevelopmentSection.vue:38]: i have fixed that number only not width @cp-mansi-d

**Review by cp-mansi-d — APPROVED** [review]

**Review by cp-jagruti-a — APPROVED** [review]

**Pull request merged**